README.md

```diff
@@ -6,10 +6,8 @@ Placeholder for the Metadata Query Service
 
 # Query Service
 [![Test Coverage](https://codecov.io/gh/HumanCellAtlas/query-service/branch/master/graph/badge.svg)](https://codecov.io/gh/HumanCellAtlas/query-service)
-
-######TODO add these links
-[![Production Health Check](health check badge SVG)](URL to service)
-[![Master Deployment Status](gitlab pipeline badge SVG)](https://allspark.dev.data.humancellatlas.org/HumanCellAtlas/query-service/pipelines)
+[![Production Health Check](health check badge SVG)](https://status.data.humancellatlas.org/service/query-service-prod.svg)
+[![Master Build Status](gitlab pipeline badge SVG)](https://allspark.dev.data.humancellatlas.org/HumanCellAtlas/query-service/pipelines)
```

Comment:
You can simply add the link I added in the pervious code review to the inner parens. When the problem is fixed with gitlab, it will light up.